website/webpack.config.js

```diff
@@ -255,6 +256,7 @@ module.exports = Object.assign({
       // I assume the `require(...)` call "succeeds" because 'fs' is shimmed to
       // be empty below.
       /flow-parser\/flow_parser\.js/,
+      /prettier/,
```

Comment:
this breaks the runtime (`require` is not defined error) but not having it messes the uglifier.

Reply:
Yeah, this has been an issue for a while. The runtime is already broken anyway. I have to look into this more. I'll merge this locally first later today.